src/frontend/utils/dato.tsx

```diff
@@ -11,58 +23,59 @@ import {
 import { PlainTekst } from '../typer/kontrakt/generelle';
 import { IFormateringsfeilmeldingerTekstinnhold } from '../typer/sanity/tekstInnhold';
 
-export const erDatoFormatGodkjent = (verdi: string) => {
-    /*FamilieDatoVelger har allerede sin egen validering.
-      Dersom valideringen går igjennom der, blir datoen formatert til YYYY-MM-DD.
-      Derfor sjekker vi her om FamilieDatoVelger har klart å formatere den,
-      i tillegg til om det er en gyldig dato med dayjs.*/
-    return dayjs(verdi, 'YYYY-MM-DD').format('YYYY-MM-DD') === verdi;
-};
+export const erDatoFormatGodkjent = (dato: Date) => isValid(dato);
 
-export const erDatoFremITid = (dato: ISODateString) => {
-    return dayjs(dato).isAfter(dayjs());
-};
+export const erDatoFremITid = (dato: Date) => isFuture(dato);
 
-export const erDatoEtterSluttdatoAvgresning = (dato: ISODateString, sluttdato: ISODateString) => {
-    return dayjs(dato).isAfter(dayjs(sluttdato));
-};
+export const erDatoEtterSluttdatoAvgresning = (dato: Date, sluttdato: Date) =>
+    isAfter(dato, sluttdato);
 
-export const erDatoFørStartDatoAvgrensning = (dato: ISODateString, startdato: ISODateString) => {
-    return dayjs(dato).isBefore(dayjs(startdato));
-};
+export const erDatoFørStartDatoAvgrensning = (dato: Date, startdato: Date) =>
+    isBefore(dato, startdato);
+
+export const gårsdagensDato = () => sub(dagensDato(), { days: 1 });
 
-export const gårsdagensDato = () => dayjs().subtract(1, 'day').format('YYYY-MM-DD');
+export const ettÅrTilbakeDato = () => sub(dagensDato(), { years: 1 });
 
-export const ettÅrTilbakeDato = () => dayjs().subtract(1, 'year').format('YYYY-MM-DD');
+export const dagensDato = () => startOfToday();
 
-export const dagensDato = () => dayjs().format('YYYY-MM-DD');
+export const morgendagensDato = () => add(dagensDato(), { days: 1 });
 
-export const morgendagensDato = () => dayjs().add(1, 'day').format('YYYY-MM-DD');
+export const erSammeDatoSomDagensDato = (dato: Date) => isToday(dato);
 
-export const erSammeDatoSomDagensDato = (dato: ISODateString) => dayjs(dato).isSame(dayjs(), 'day');
+export const dagenEtterDato = (dato: Date) => add(dato, { days: 1 });
 
-export const dagenEtterDato = (dato: ISODateString) =>
-    dayjs(dato).add(1, 'day').format('YYYY-MM-DD');
+export const tidenesMorgen = () => startOfDay(new Date(1000, 0));
+
+export const tidenesEnde = () => startOfDay(new Date(3000, 0));
+
+export const stringTilDate = (dato: string) => startOfDay(new Date(dato));
+
+export const parseTilGyldigDato = (dateString: string): Date | undefined => {
+    const parsetDato = parse(dateString, 'dd.MM.yyyy', new Date());
+    const parsetDatoErGyldig = erDatoFormatGodkjent(parsetDato);
+    return parsetDatoErGyldig ? parsetDato : undefined;
+};
```

Comment:
Denne metoden gir oss alltid `Invalid Date`, da `dateString` ikke er på formatet 'dd.MM.yyyy'. Formatet på dateString er 'ISO' pga `formatISO` når vi setter dato-feltet. Testet lokalt hos meg og da fungerte det med: 
```suggestion
export const parseTilGyldigDato = (dateString: string): Date | undefined => {
    const parsetDato = parseISO(dateString);
    const parsetDatoErGyldig = erDatoFormatGodkjent(parsetDato);
    return parsetDatoErGyldig ? parsetDato : undefined;
};
```